api/types/constants.go

```diff
@@ -546,6 +546,10 @@ const (
 	// a managed device.
 	KindAccessGraphSecretPrivateKey = "access_graph_private_key"
 
+	// KindProvisionigState is a resource that tracks provisioning of a user or access
```

Comment:
```suggestion
	// KindProvisioningState is a resource that tracks provisioning of a user or access
```

Reply:
I am apparently biologically incapable of typeing the word "provisioning" without missing at least one 'n'.